keycloak/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM jboss/keycloak-postgres:2.4.0.Final
+FROM jboss/keycloak-postgres:2.5.0.Final
 
 ENV MAVEN_VER=3.3.9
 ENV MAVEN_DIST=apache-maven-$MAVEN_VER
 ENV MAVEN_FILE=$MAVEN_DIST-bin.tar.gz
 ENV PATH=$PATH:/tmp/$MAVEN_DIST/bin
 ENV KC_SPI_SRC=providers
 ENV KC_SPI_DEST=/usr/src/keycloak
+ENV KC_VER=2.4.0.Final
```

Comment:
This is the issue I mentioned on https://github.com/cfpb/hmda-platform-auth/pull/49#issue-198604487.  For whatever reason, they never published a `2.5.x` of the Keycloak Java libs.  `2.4.0.Final` libs _seem_ to work fine.

Perhaps I should put in a comment about that, and change the variable name to `KC_LIB_VER`.